gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -541,7 +541,14 @@ gst_tensor_trainer_chain (GstPad * sinkpad, GstObject * parent,
   if (trainer->total_invoke_num == 1
       || trainer->total_invoke_num ==
       trainer->prop.num_train_samples + trainer->prop.num_valid_samples) {
-
+#if 0
```

Comment:
plz remove these block.

Reply:
okay, thank you.

Reply:
Done ([c9511e6](https://github.com/nnstreamer/nnstreamer/pull/4023/commits/c9511e6d1a5b12ae09ffa85ac5598e8457bc01fa))
Thank you.